src/halboy/navigator.clj

```diff
@@ -181,6 +191,15 @@
   ([resource settings]
    (resource->Navigator resource settings)))
 
+(defn head
+  ([navigator link]
```

Comment:
Can you add a comment to this method, so it is in line with other methods?

@ChloeAcheampong @tobyclemson

Reply:
They all have descriptions that aren't very HTTP oriented. We were going to go with, "Performs a HEAD request against a link in an API.". Does that work for you? Anything more in line with the others?

Reply:
That would be great.

Reply:
Done.